### DISCUSSION TITLE
Display "Add Host Manually" on the moonlight on the ios device

### DISCUSSION THREAD

**Discussion by nini45678** [LizardByte/community#160] [Sunshine]
Computer users from other countries say hello to your team！
At present, I cannot add the computer by manually entering the computer ip on my ios device, but it can be added to this computer on another Samsung note10+ and can be accessed。
After changing to ipad pro, I can’t manually add a computer, but the problem 
![65456](https://user-images.githubusercontent.com/78917038/216903512-7f24535f-b8e4-4527-aae4-e06921090426.jpg)

**Comment by xzcxzcxzc333** [comment]
Hello, has the problem been solved? When I installed **moonlight** on my iPad Air 5, I couldn't connect to **sunshine** on my computer o(╥﹏╥)o

[a comment by an automated account is not shown]
